vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SlideManager.java

```diff
@@ -41,4 +41,11 @@ public ISlide getSlide(String slideId) {
         }
         return null;
     }
+
+    @Override
+    public ISlide updateSlide(Slide slide) {
+        slideRepo.save((Slide) slide);
+        return null;
```

Comment:
unless your need the method to return something, don't return anything. always returning null doesn't really make sense.